### ISSUE TITLE
Pull&Push command

### ISSUE THREAD

**Issue by antonakv** [SublimeGit/SublimeGit#180]
Hi!

How can I add Pull&Push to commands list ? It's more convenient than every time call Pull and then call Push.
Thank you.

**Comment by AlessandroCalo** [comment]
@antonakv there can always be conflicts after pulling, which you need to look at and fix manually, so IMHO it wouldn't be wise to push automatically right after..

**Comment by miped** [comment]
I don't think this is a good command to have in core. Perhaps an implementation of #92 should support multiple commands in sequence.
